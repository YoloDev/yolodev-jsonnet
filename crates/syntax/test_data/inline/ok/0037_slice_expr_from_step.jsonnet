foo[0::2]
