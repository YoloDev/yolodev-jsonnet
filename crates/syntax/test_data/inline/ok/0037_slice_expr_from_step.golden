SOURCE_FILE@0..10
  SLICE_EXPR@0..9
    IDENT_EXPR@0..3
      IDENT@0..3 "foo"
    L_BRACK@3..4 "["
    SLICE_FROM@4..5
      NUMBER_EXPR@4..5
        NUMBER@4..5 "0"
    DOUBLE_COLON_OP@5..7 "::"
    SLICE_STEP@7..8
      NUMBER_EXPR@7..8
        NUMBER@7..8 "2"
    R_BRACK@8..9 "]"
  WHITESPACE@9..10 "\n"
